### DISCUSSION TITLE
Why is taking so much system memory?

### DISCUSSION THREAD

**Discussion by osmancitov** [Tyrrrz/DiscordChatExporter#1257] [Q&A]
DiscrodChatExporter is taking around 6GB of physical and system commit, virtual memory.
This looks to me like an anomaly of some sort.
Can somebody tell me something about this?
Is this normal?
I attach a picture, where it shows a big dent, like 3 o 4GB of memory,
with just closing and re-open it again.
![big dent](https://github.com/Tyrrrz/DiscordChatExporter/assets/13158838/a4f29f9c-42ae-4723-a550-993cac233069)

**Comment by Tyrrrz** [comment]
Does it happen after you try to export a channel? Or immediately on startup?
